carbonmark-api/test/routes/listings/[id]/get.test.ts

```diff
@@ -0,0 +1,38 @@
+import { FastifyInstance } from "fastify";
+import nock from "nock";
+import { GRAPH_URLS } from "../../../../src/app.constants";
+import { fixtures } from "../../../fixtures";
+import { build } from "../../../helper";
+import { DEV_URL } from "../../../test.constants";
+
+describe("GET /listings/:id", () => {
+  let fastify: FastifyInstance;
+
+  // Setup the server
+  beforeEach(async () => {
+    try {
+      fastify = await build();
+    } catch (e) {
+      console.error("/listings/:id get.test.ts setup failed", e);
+    }
+  });
+  test("Returns listing with token symbol", async () => {
+    nock(GRAPH_URLS["polygon"].marketplace)
+      .post("")
+      .reply(200, {
+        data: { listing: fixtures.marketplace.listing },
+      });
+    nock(GRAPH_URLS["polygon"].digitalCarbon)
```

Comment:
@0xemc puts mocks in a get.test.mocks.ts file (see projects for instance). I think kit would be good to generalize this practice here.

Reply:
Done 👍